Source/WebCore/css/CSSPrimitiveValue.cpp

```diff
@@ -838,16 +838,16 @@ double CSSPrimitiveValue::computeNonCalcLengthDouble(const CSSToLengthConversion
     switch (primitiveType) {
     case CSSUnitType::CSS_EMS:
     case CSSUnitType::CSS_QUIRKY_EMS:
-        ASSERT(conversionData.style());
-        value = computeUnzoomedNonCalcLengthDouble(primitiveType, value, conversionData.propertyToCompute(), nullptr, &conversionData.style()->fontDescription());
+        ASSERT(conversionData.styleForFontUnits());
```

Comment:
Can we actually guarantee that `styleForFontUnits()` is valid (i.e. that we have a `parentStyle()`) no matter how we get here?